src/ucp/core/ucp_worker.c

```diff
@@ -1960,11 +1960,11 @@ void ucp_worker_destroy(ucp_worker_h worker)
     UCS_ASYNC_UNBLOCK(&worker->async);
 
     ucp_worker_destroy_ep_configs(worker);
+    ucp_tag_match_cleanup(&worker->tm);
+    ucp_worker_close_ifaces(worker);
```

Comment:
why need to move ucp_worker_close_ifaces()?

Reply:
to release `rdesc`s to UCT before UCT iface is closed

Reply:
but why ucp_worker_close_ifaces() has to be moved before ucs_mpool_cleanup calls?
can just move ucp_tag_match_cleanup

Reply:
ah, I see
fixed